better-custom-response/custom-responses.ts

```diff
@@ -118,6 +118,22 @@ const customResponses: CustomResponse[] = [
         icon_emoji: ':kadokawa:',
         username: 'KADOKAWA',
     },
+    {
+        input: [/デニム/],
+        outputFunction: (input: string[]) => {
+            const lane = shuffle([1, 2, 3, 4, 5, 6, 7]);
+            let white = "|　　|";
+            let black = "|　　|";
+            for(let i = 0; i < 7; i++){
+                white += ((lane[i] % 2 === 1) ? "＿|" : "　|");
+                black += ((lane[i] % 2 === 0) ? "＿|" : "　|");
+            }
```

Comment:
```suggestion
            const white = "|　　|" + lane.map((i: number) => (i % 2 == 1) ? "＿|" : "　|").join("");
            const black = "|　　|" + lane.map((i: number) => (i % 2 == 0) ? "＿|" : "　|").join("");
```
constを食べたかったから注文したの！

Reply:
頭悪いので等号ガバガバんなっちゃった

Reply:
勝手にeditしました

Reply:
天才？